airflow/providers/oracle/hooks/oracle.py

```diff
@@ -90,6 +152,40 @@ def get_conn(self) -> oracledb.Connection:
         mod = conn.extra_dejson.get('module')
         schema = conn.schema
 
+        # Enable oracledb thick mode if thick_mode is set to True
+        # Parameters take precedence over connection config extra
+        # Defaults to use thin mode if not provided in params or connection config extra
+        thick_mode = _get_first_bool(self.thick_mode, conn.extra_dejson.get('thick_mode'))
+        if thick_mode is True:
+            if self.thick_mode_lib_dir is None:
+                self.thick_mode_lib_dir = conn.extra_dejson.get('thick_mode_lib_dir')
+                if not isinstance(self.thick_mode_lib_dir, (str, type(None))):
+                    raise TypeError(
+                        f'thick_mode_lib_dir expected str or None, '
```

Comment:
```suggestion
                        'thick_mode_lib_dir expected str or None, '
```
Nit: f-strings not needed here.

Reply:
I thought it was needed on every line when you do a multiline f-string.  Turns out I was wrong about that! :)
It looks like it doesn't make a difference for plain strings though:  https://stackoverflow.com/questions/48471086/style-for-continued-multi-line-f-strings
Do we have a style guide for this?  To me prepending each line with `f` looks cleaner.

Reply:
Nope, just a nit. I have no strong feelings really. 🙂